src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocNodeImpl.java

```diff
@@ -94,7 +95,9 @@ public int getColumnNumber() {
 
     @Override
     public DetailNode[] getChildren() {
-        if (children == null) {
+        if (children == null
+                // remove excess tokens
+                || type == JavadocTokenTypes.TEXT) {
```

Comment:
I do not like such hack. 
We need to change tree right after parse. JavadocNodeImpl should stay simple POJO data container.